docs/api/apiv3/paths/work_package_reminders.yml

```diff
@@ -1,61 +1,139 @@
 # /api/v3/work_packages/{id}/reminders
 ---
 get:
+  summary: List work package reminders
+  operationId: list_reminders
+  tags:
+    - Work Packages
+    - Reminders
+  description: |-
+    Gets a list of your upcoming reminders for this work package.
```

Comment:
I've read the newly added error message before: Is it true that the "list" of reminders will either be empty or contain exactly one reminder, but never two?

One additional thought: How useful would it be to list reminders independent of a work package? I.e. `/api/v3/reminders`. This could be a way to find _all_ reminders I have set.

Reply:
> I've read the newly added error message before: Is it true that the "list" of reminders will either be empty or contain exactly one reminder, but never two?

Yes, this is true. Only one active reminder is allowed at the moment. Let me touch up the wording to better reflect that. 👍🏾 

> One additional thought: How useful would it be to list reminders independent of a work package? I.e. `/api/v3/reminders`. This could be a way to find _all_ reminders I have set.

This would be useful, and it's something we have planned for the future but out of scope of the current work.

Reply:
> it's something we have planned for the future but out of scope of the current work.

Is there already a work package for the future work? Looking at the work package linked, I just saw that "reading" own reminders is a requirement, but there was neither out-of-scope mentioned, nor a work package for the future work.

Let's make sure it's written down, if we plan on extending it this way :)

Reply:
Implemented separately: https://github.com/opf/openproject/pull/19083